src/System.IO.Compression.Brotli/System/IO/Compression/Brotli.cs

```diff
@@ -117,6 +118,10 @@ internal static void EnsureInitialized(ref State state, bool compress)
 
         public static int GetMaximumCompressedSize(int inputSize)
         {
+            if (inputSize < 0 || inputSize > MaxInputSize)
```

Comment:
Aside from integer overflow when doing:
```C#
int result = inputSize + overhead;
```
when will `(result < inputSize)` ever be true in `return (result < inputSize) ? inputSize : result;`?

And since you are now catching the overflow case here (`if (inputSize < 0 || inputSize > MaxInputSize)`), do we need to check if result will be less than inputSize?